### PR TITLE
EIS-343 - Using ReactModal

### PR DESCRIPTION
**Pull request by lortiz-daitan**
- Uses ReactModal
- Set modal root element
- Keep the focus inside the wizard (on next button) to make the modal always listen to ESC key
- Cross button refactor to icon and outline

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -68,7 +68,9 @@
     "@tippy.js/react": "^3.1.1",
     "classnames": "^2.2.6",
     "copy-to-clipboard": "^3.2.0",
-    "downshift": "^4.0.8"
+    "downshift": "^4.0.8",
+    "jest-css-modules": "^2.1.0",
```
Comment (tecneider) [inline package.json]: jest-css-modules is a dev dependency can you move it - otherwise eslint will not allow it in production files

**Comment by tecneider** [comment]
Checked in storybook - works well - just change the dev dependency position

**Comment by tecneider** [comment]
Screenshot 2020-02-06 at 15 25 36

Can you hide the outline?

**Comment by lortiz-daitan** [comment]
> Screenshot 2020-02-06 at 15 25 36
> 
> Can you hide the outline?

I could add an outline none, but it's not a good pratice.
Some extension apps (Giltlab, PagerDutty, Confluence) have a mechanism of showing the outline on "tab" and hide it on click. I think we should open a task to implement this mechanism

File: package.json (hunk 1 above)
Comment (lortiz-daitan) [reply package.json]: Sure. I just noticed that it's already there. It's duplicated.

**Pull request merged**